src-vue/src/views/ChangelogView.vue

```diff
@@ -1,24 +1,78 @@
 <template>
-    <div class="fc__changelog__container">
-        <el-link :underline="false" icon="DocumentCopy" target="_blank"
-            href="https://github.com/R2Northstar/Northstar/releases">
-            Open release notes
-        </el-link>
+    <div style="height: calc(100% - var(--fc-menu_height))">
+        <div v-if="releases.length === 0" class="fc__changelog__container">
+            <el-progress :show-text="false" :percentage="50" :indeterminate="true" />
```

Comment:
The loading bar was fake all along? :O

Reply:
What do you mean, "fake"?
It's only there to say some fetching is occuring :)

Reply:
"Fake" as in it's not updated based on how much content has been downloaded :P

Reply:
Nah, since we don't see the progress bar that much, I don't think it's worth the development effort

Reply:
Yeah that's totally fine. Didn't mean to imply it should be proper. Not worth the effort as you said :P